[2025-03-27T00:13:23.907+0000] {processor.py:157} INFO - Started process (PID=622) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:23.908+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:13:23.915+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:23.915+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:23.931+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:13:23.952+0000] {processor.py:836} INFO - DAG(s) dict_keys(['postgres_to_bigquery_transfer']) retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:24.344+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:24.343+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2025-03-27T00:13:24.358+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:24.357+0000] {dag.py:3490} INFO - Setting next_dagrun for postgres_to_bigquery_transfer to None, run_after=None
[2025-03-27T00:13:24.383+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.483 seconds
[2025-03-27T00:13:55.209+0000] {processor.py:157} INFO - Started process (PID=642) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:55.210+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:13:55.215+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:55.215+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:55.233+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:13:55.253+0000] {processor.py:836} INFO - DAG(s) dict_keys(['postgres_to_bigquery_transfer']) retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:13:55.309+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:55.309+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2025-03-27T00:13:55.353+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:13:55.353+0000] {dag.py:3490} INFO - Setting next_dagrun for postgres_to_bigquery_transfer to None, run_after=None
[2025-03-27T00:13:55.373+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.170 seconds
[2025-03-27T00:14:01.406+0000] {processor.py:157} INFO - Started process (PID=647) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:01.409+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:14:01.413+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:01.413+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:01.458+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:14:01.489+0000] {processor.py:836} INFO - DAG(s) dict_keys(['postgres_to_bigquery_transfer']) retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:01.605+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:01.603+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2025-03-27T00:14:01.678+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:01.678+0000] {dag.py:3490} INFO - Setting next_dagrun for postgres_to_bigquery_transfer to None, run_after=None
[2025-03-27T00:14:01.701+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.307 seconds
[2025-03-27T00:14:04.783+0000] {processor.py:157} INFO - Started process (PID=652) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:04.785+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:14:04.790+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:04.790+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:04.812+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:14:04.841+0000] {processor.py:836} INFO - DAG(s) dict_keys(['postgres_to_bigquery_transfer']) retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:04.916+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:04.916+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2025-03-27T00:14:04.944+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:04.944+0000] {dag.py:3490} INFO - Setting next_dagrun for postgres_to_bigquery_transfer to None, run_after=None
[2025-03-27T00:14:04.972+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.195 seconds
[2025-03-27T00:14:35.639+0000] {processor.py:157} INFO - Started process (PID=672) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:35.640+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:14:35.647+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:35.646+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:35.659+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:14:35.683+0000] {processor.py:836} INFO - DAG(s) dict_keys(['postgres_to_bigquery_transfer']) retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:35.715+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:35.715+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2025-03-27T00:14:35.737+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:35.737+0000] {dag.py:3490} INFO - Setting next_dagrun for postgres_to_bigquery_transfer to None, run_after=None
[2025-03-27T00:14:35.757+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.128 seconds
[2025-03-27T00:14:44.838+0000] {processor.py:157} INFO - Started process (PID=677) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:44.842+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:14:44.846+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:44.846+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:44.853+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:44.851+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 85
    load_task = GCSToBigQueryOperator(
    ^
IndentationError: unexpected indent
[2025-03-27T00:14:44.853+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:44.905+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:14:50.025+0000] {processor.py:157} INFO - Started process (PID=682) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:50.026+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:14:50.030+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:50.030+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:50.051+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:14:50.056+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:14:50.054+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 75, in <module>
    postgres_conn_id=POSTGRES_CONN,
NameError: name 'POSTGRES_CONN' is not defined
[2025-03-27T00:14:50.056+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:14:50.087+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:15:05.700+0000] {processor.py:157} INFO - Started process (PID=702) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:05.702+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:15:05.706+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:15:05.706+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:05.732+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:15:05.737+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:15:05.735+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:15:05.737+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:05.792+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.110 seconds
[2025-03-27T00:15:36.598+0000] {processor.py:157} INFO - Started process (PID=722) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:36.601+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:15:36.605+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:15:36.605+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:36.621+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:15:36.626+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:15:36.624+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:15:36.627+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:15:36.662+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.068 seconds
[2025-03-27T00:16:07.672+0000] {processor.py:157} INFO - Started process (PID=742) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:07.677+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:16:07.681+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:16:07.681+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:07.704+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:16:07.715+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:16:07.707+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:16:07.716+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:07.767+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.108 seconds
[2025-03-27T00:16:38.645+0000] {processor.py:157} INFO - Started process (PID=762) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:38.648+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:16:38.656+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:16:38.655+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:38.690+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:16:38.702+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:16:38.697+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:16:38.703+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:16:38.766+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.132 seconds
[2025-03-27T00:17:09.406+0000] {processor.py:157} INFO - Started process (PID=782) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:09.410+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:17:09.414+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:17:09.414+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:09.434+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:17:09.442+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:17:09.437+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:17:09.442+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:09.498+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.102 seconds
[2025-03-27T00:17:40.522+0000] {processor.py:157} INFO - Started process (PID=802) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:40.523+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:17:40.529+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:17:40.529+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:40.543+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:17:40.548+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:17:40.546+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:17:40.549+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:17:40.609+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.093 seconds
[2025-03-27T00:18:11.542+0000] {processor.py:157} INFO - Started process (PID=822) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:11.544+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:18:11.549+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:18:11.549+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:11.569+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:18:11.574+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:18:11.572+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:18:11.575+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:11.607+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:18:41.805+0000] {processor.py:157} INFO - Started process (PID=842) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:41.807+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:18:41.810+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:18:41.810+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:41.825+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:18:41.830+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:18:41.827+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:18:41.830+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:18:41.868+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.068 seconds
[2025-03-27T00:19:12.110+0000] {processor.py:157} INFO - Started process (PID=862) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:12.113+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:19:12.118+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:19:12.118+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:12.144+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:19:12.152+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:19:12.147+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:19:12.152+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:12.206+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.106 seconds
[2025-03-27T00:19:43.168+0000] {processor.py:157} INFO - Started process (PID=882) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:43.170+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:19:43.174+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:19:43.174+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:43.190+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:19:43.251+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:19:43.193+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:19:43.252+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:19:43.307+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.145 seconds
[2025-03-27T00:20:14.040+0000] {processor.py:157} INFO - Started process (PID=902) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:14.041+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:20:14.049+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:20:14.049+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:14.064+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:20:14.068+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:20:14.066+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:20:14.068+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:14.107+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.072 seconds
[2025-03-27T00:20:44.918+0000] {processor.py:157} INFO - Started process (PID=922) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:44.920+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:20:44.925+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:20:44.925+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:44.943+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:20:44.949+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:20:44.946+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:20:44.950+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:20:44.994+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.089 seconds
[2025-03-27T00:21:15.122+0000] {processor.py:157} INFO - Started process (PID=942) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:15.125+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:21:15.130+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:21:15.130+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:15.155+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:21:15.159+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:21:15.157+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:21:15.160+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:15.212+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.102 seconds
[2025-03-27T00:21:46.043+0000] {processor.py:157} INFO - Started process (PID=962) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:46.045+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:21:46.048+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:21:46.048+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:46.069+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:21:46.073+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:21:46.071+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:21:46.074+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:21:46.110+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.072 seconds
[2025-03-27T00:22:16.816+0000] {processor.py:157} INFO - Started process (PID=982) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:16.818+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:22:16.823+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:22:16.822+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:16.837+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:22:16.843+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:22:16.840+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:22:16.843+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:16.873+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.065 seconds
[2025-03-27T00:22:47.568+0000] {processor.py:157} INFO - Started process (PID=1002) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:47.575+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:22:47.580+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:22:47.580+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:47.593+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:22:47.602+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:22:47.600+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:22:47.602+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:22:47.636+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.071 seconds
[2025-03-27T00:23:18.837+0000] {processor.py:157} INFO - Started process (PID=1022) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:18.839+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:23:18.843+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:23:18.843+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:18.865+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:23:18.870+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:23:18.867+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:23:18.870+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:18.913+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.086 seconds
[2025-03-27T00:23:49.110+0000] {processor.py:157} INFO - Started process (PID=1042) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:49.112+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:23:49.116+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:23:49.116+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:49.128+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:23:49.136+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:23:49.133+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:23:49.136+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:23:49.191+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.088 seconds
[2025-03-27T00:24:20.172+0000] {processor.py:157} INFO - Started process (PID=1062) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:20.176+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:24:20.181+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:24:20.180+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:20.195+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:24:20.199+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:24:20.197+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:24:20.199+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:20.236+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.069 seconds
[2025-03-27T00:24:50.837+0000] {processor.py:157} INFO - Started process (PID=1082) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:50.838+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:24:50.847+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:24:50.847+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:50.859+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:24:50.913+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:24:50.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:24:50.914+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:24:50.965+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.134 seconds
[2025-03-27T00:25:21.914+0000] {processor.py:157} INFO - Started process (PID=1102) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:21.915+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:25:21.919+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:25:21.919+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:21.930+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:25:21.936+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:25:21.932+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:25:21.937+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:21.965+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.057 seconds
[2025-03-27T00:25:52.609+0000] {processor.py:157} INFO - Started process (PID=1122) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:52.611+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:25:52.617+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:25:52.617+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:52.636+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:25:52.641+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:25:52.638+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:25:52.641+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:25:52.677+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:26:23.377+0000] {processor.py:157} INFO - Started process (PID=1142) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:23.379+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:26:23.383+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:26:23.383+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:23.400+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:26:23.404+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:26:23.402+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:26:23.405+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:23.441+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.071 seconds
[2025-03-27T00:26:54.267+0000] {processor.py:157} INFO - Started process (PID=1162) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:54.268+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:26:54.280+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:26:54.280+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:54.292+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:26:54.300+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:26:54.295+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:26:54.300+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:26:54.340+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.078 seconds
[2025-03-27T00:27:24.495+0000] {processor.py:157} INFO - Started process (PID=1182) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:24.497+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:27:24.504+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:27:24.504+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:24.520+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:27:24.524+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:27:24.522+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:27:24.525+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:24.554+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.069 seconds
[2025-03-27T00:27:55.556+0000] {processor.py:157} INFO - Started process (PID=1202) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:55.557+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:27:55.561+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:27:55.561+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:55.572+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:27:55.578+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:27:55.574+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:27:55.578+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:27:55.620+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:28:26.740+0000] {processor.py:157} INFO - Started process (PID=1222) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:26.742+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:28:26.745+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:28:26.745+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:26.764+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:28:26.770+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:28:26.767+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:28:26.771+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:26.806+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.078 seconds
[2025-03-27T00:28:57.693+0000] {processor.py:157} INFO - Started process (PID=1242) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:57.695+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:28:57.699+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:28:57.699+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:57.711+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:28:57.715+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:28:57.713+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:28:57.715+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:28:57.753+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.069 seconds
[2025-03-27T00:29:28.141+0000] {processor.py:157} INFO - Started process (PID=1262) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:28.142+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:29:28.146+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:29:28.146+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:28.163+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:29:28.167+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:29:28.165+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:29:28.168+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:28.202+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.066 seconds
[2025-03-27T00:29:58.911+0000] {processor.py:157} INFO - Started process (PID=1282) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:58.913+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:29:58.917+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:29:58.917+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:58.935+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:29:58.939+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:29:58.937+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:29:58.939+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:29:58.970+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.064 seconds
[2025-03-27T00:30:29.755+0000] {processor.py:157} INFO - Started process (PID=1302) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:30:29.757+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:30:29.761+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:30:29.761+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:30:29.775+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:30:29.815+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:30:29.777+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:30:29.816+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:30:30.061+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.311 seconds
[2025-03-27T00:31:00.871+0000] {processor.py:157} INFO - Started process (PID=1322) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:00.875+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:31:00.880+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:31:00.880+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:00.894+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:31:00.903+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:31:00.897+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:31:00.904+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:00.936+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.083 seconds
[2025-03-27T00:31:31.965+0000] {processor.py:157} INFO - Started process (PID=1342) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:31.967+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:31:31.971+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:31:31.971+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:31.990+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:31:31.994+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:31:31.992+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:31:31.995+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:31:32.049+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.092 seconds
[2025-03-27T00:32:03.069+0000] {processor.py:157} INFO - Started process (PID=1362) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:03.070+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:32:03.075+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:32:03.074+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:03.094+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:32:03.098+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:32:03.096+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:32:03.098+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:03.133+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.069 seconds
[2025-03-27T00:32:34.047+0000] {processor.py:157} INFO - Started process (PID=1382) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:34.050+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:32:34.062+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:32:34.062+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:34.078+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:32:34.086+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:32:34.083+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:32:34.086+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:32:34.121+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.084 seconds
[2025-03-27T00:33:04.947+0000] {processor.py:157} INFO - Started process (PID=1402) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:04.949+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:33:04.954+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:33:04.954+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:04.974+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:33:04.979+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:33:04.977+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:33:04.979+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:05.016+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.076 seconds
[2025-03-27T00:33:35.413+0000] {processor.py:157} INFO - Started process (PID=1422) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:35.416+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:33:35.428+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:33:35.427+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:35.448+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:33:35.462+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:33:35.453+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:33:35.464+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:33:35.514+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.109 seconds
[2025-03-27T00:34:06.545+0000] {processor.py:157} INFO - Started process (PID=1442) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:06.548+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:34:06.556+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:34:06.556+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:06.586+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:34:06.596+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:34:06.593+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:34:06.597+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:06.667+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.133 seconds
[2025-03-27T00:34:37.624+0000] {processor.py:157} INFO - Started process (PID=1462) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:37.626+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:34:37.631+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:34:37.630+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:37.650+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:34:37.655+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:34:37.653+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:34:37.656+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:34:37.693+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.075 seconds
[2025-03-27T00:35:08.393+0000] {processor.py:157} INFO - Started process (PID=1482) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:08.395+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:35:08.399+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:35:08.399+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:08.412+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:35:08.421+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:35:08.419+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:35:08.422+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:08.462+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.079 seconds
[2025-03-27T00:35:39.221+0000] {processor.py:157} INFO - Started process (PID=1502) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:39.223+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:35:39.229+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:35:39.229+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:39.252+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:35:39.258+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:35:39.254+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:35:39.259+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:35:39.342+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.126 seconds
[2025-03-27T00:36:10.140+0000] {processor.py:157} INFO - Started process (PID=1522) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:10.141+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:36:10.146+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:36:10.146+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:10.165+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:36:10.170+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:36:10.168+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:36:10.170+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:10.209+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.075 seconds
[2025-03-27T00:36:41.055+0000] {processor.py:157} INFO - Started process (PID=1542) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:41.058+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:36:41.074+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:36:41.074+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:41.087+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:36:41.091+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:36:41.089+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:36:41.092+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:36:41.130+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.081 seconds
[2025-03-27T00:37:11.972+0000] {processor.py:157} INFO - Started process (PID=1562) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:11.974+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:37:11.983+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:37:11.980+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:12.011+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:37:12.017+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:37:12.014+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:37:12.018+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:12.058+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.095 seconds
[2025-03-27T00:37:42.870+0000] {processor.py:157} INFO - Started process (PID=1582) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:42.872+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:37:42.877+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:37:42.877+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:42.895+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:37:42.899+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:37:42.897+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:37:42.900+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:37:42.935+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.076 seconds
[2025-03-27T00:38:13.991+0000] {processor.py:157} INFO - Started process (PID=1602) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:13.993+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:38:14.001+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:38:14.001+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:14.015+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:38:14.020+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:38:14.017+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:38:14.020+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:14.062+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.077 seconds
[2025-03-27T00:38:44.542+0000] {processor.py:157} INFO - Started process (PID=1622) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:44.544+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:38:44.548+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:38:44.548+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:44.560+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:38:44.569+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:38:44.563+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:38:44.569+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:38:44.607+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.073 seconds
[2025-03-27T00:39:15.497+0000] {processor.py:157} INFO - Started process (PID=1642) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:15.498+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:39:15.511+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:39:15.510+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:15.523+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:39:15.527+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:39:15.526+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:39:15.528+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:15.561+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.073 seconds
[2025-03-27T00:39:46.557+0000] {processor.py:157} INFO - Started process (PID=1662) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:46.568+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:39:46.573+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:39:46.573+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:46.592+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:39:46.597+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:39:46.595+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:39:46.597+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:39:46.651+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.108 seconds
[2025-03-27T00:40:16.928+0000] {processor.py:157} INFO - Started process (PID=1682) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:16.930+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:40:16.937+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:40:16.936+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:16.955+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:40:16.959+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:40:16.957+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:40:16.960+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:16.991+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.084 seconds
[2025-03-27T00:40:47.997+0000] {processor.py:157} INFO - Started process (PID=1702) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:48.000+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:40:48.007+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:40:48.007+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:48.029+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:40:48.035+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:40:48.032+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:40:48.036+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:40:48.078+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.088 seconds
[2025-03-27T00:41:19.042+0000] {processor.py:157} INFO - Started process (PID=1722) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:19.043+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:41:19.050+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:41:19.050+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:19.067+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:41:19.073+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:41:19.070+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:41:19.073+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:19.108+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.074 seconds
[2025-03-27T00:41:49.941+0000] {processor.py:157} INFO - Started process (PID=1742) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:49.942+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:41:49.953+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:41:49.952+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:49.972+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:41:49.977+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:41:49.975+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:41:49.977+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:41:50.019+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.083 seconds
[2025-03-27T00:42:20.729+0000] {processor.py:157} INFO - Started process (PID=1762) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:20.731+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:42:20.738+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:42:20.737+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:20.757+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:42:20.762+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:42:20.760+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:42:20.762+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:20.793+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.069 seconds
[2025-03-27T00:42:51.928+0000] {processor.py:157} INFO - Started process (PID=1782) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:51.930+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:42:51.937+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:42:51.937+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:51.961+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:42:51.970+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:42:51.967+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:42:51.972+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:42:52.017+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.106 seconds
[2025-03-27T00:43:22.834+0000] {processor.py:157} INFO - Started process (PID=1802) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:22.835+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:43:22.841+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:43:22.841+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:22.864+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:43:22.870+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:43:22.867+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:43:22.870+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:22.909+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.083 seconds
[2025-03-27T00:43:53.938+0000] {processor.py:157} INFO - Started process (PID=1832) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:53.940+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:43:53.946+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:43:53.946+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:53.966+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:43:53.973+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:43:53.970+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:43:53.974+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:43:54.021+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.093 seconds
[2025-03-27T00:44:25.103+0000] {processor.py:157} INFO - Started process (PID=1862) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:25.104+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:44:25.118+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:44:25.117+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:25.136+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:44:25.141+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:44:25.139+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:44:25.142+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:25.179+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.082 seconds
[2025-03-27T00:44:56.062+0000] {processor.py:157} INFO - Started process (PID=1892) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:56.063+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:44:56.071+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:44:56.070+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:56.085+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:44:56.090+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:44:56.088+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:44:56.090+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:44:56.125+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.068 seconds
[2025-03-27T00:45:27.033+0000] {processor.py:157} INFO - Started process (PID=1917) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:27.034+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:45:27.045+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:45:27.044+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:27.058+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:45:27.063+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:45:27.061+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:45:27.063+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:27.104+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.079 seconds
[2025-03-27T00:45:57.938+0000] {processor.py:157} INFO - Started process (PID=1942) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:57.940+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:45:57.944+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:45:57.944+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:57.962+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:45:57.967+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:45:57.965+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:45:57.967+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:45:58.003+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.071 seconds
[2025-03-27T00:46:28.825+0000] {processor.py:157} INFO - Started process (PID=1967) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:28.827+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:46:28.836+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:46:28.836+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:28.848+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:46:28.853+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:46:28.850+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:46:28.853+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:28.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.078 seconds
[2025-03-27T00:46:59.055+0000] {processor.py:157} INFO - Started process (PID=2002) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:59.058+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:46:59.061+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:46:59.061+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:59.072+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:46:59.075+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:46:59.074+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:46:59.075+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:46:59.092+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.046 seconds
[2025-03-27T00:47:29.276+0000] {processor.py:157} INFO - Started process (PID=2042) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:47:29.277+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:47:29.280+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:47:29.280+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:47:29.295+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:47:29.300+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:47:29.298+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:47:29.300+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:47:29.332+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.059 seconds
[2025-03-27T00:48:00.233+0000] {processor.py:157} INFO - Started process (PID=2072) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:00.235+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:48:00.239+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:48:00.239+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:00.260+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:48:00.265+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:48:00.263+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:48:00.266+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:00.349+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.129 seconds
[2025-03-27T00:48:30.503+0000] {processor.py:157} INFO - Started process (PID=2097) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:30.506+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:48:30.511+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:48:30.511+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:30.533+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:48:30.537+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:48:30.535+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:48:30.537+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:48:30.577+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.088 seconds
[2025-03-27T00:49:00.864+0000] {processor.py:157} INFO - Started process (PID=2122) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:00.867+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:49:00.878+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:49:00.878+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:00.913+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:49:00.925+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:49:00.923+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:49:00.925+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:00.981+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.133 seconds
[2025-03-27T00:49:31.618+0000] {processor.py:157} INFO - Started process (PID=2147) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:31.620+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:49:31.625+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:49:31.625+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:31.653+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:49:31.664+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:49:31.657+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:49:31.673+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:49:31.727+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.119 seconds
[2025-03-27T00:50:02.579+0000] {processor.py:157} INFO - Started process (PID=2172) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:02.581+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:50:02.585+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:50:02.585+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:02.604+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:50:02.610+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:50:02.607+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:50:02.610+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:02.647+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.078 seconds
[2025-03-27T00:50:33.623+0000] {processor.py:157} INFO - Started process (PID=2197) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:33.625+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:50:33.630+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:50:33.630+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:33.644+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:50:33.648+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:50:33.646+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:50:33.648+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:50:33.692+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.083 seconds
[2025-03-27T00:51:04.506+0000] {processor.py:157} INFO - Started process (PID=2222) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:04.508+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:51:04.512+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:51:04.512+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:04.531+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:51:04.535+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:51:04.533+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:51:04.536+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:04.573+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.073 seconds
[2025-03-27T00:51:35.249+0000] {processor.py:157} INFO - Started process (PID=2247) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:35.251+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:51:35.261+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:51:35.260+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:35.276+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:51:35.280+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:51:35.278+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:51:35.281+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:51:35.324+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.082 seconds
[2025-03-27T00:52:06.190+0000] {processor.py:157} INFO - Started process (PID=2272) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:06.217+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:52:06.243+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:52:06.241+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:06.320+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:52:06.384+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:52:06.378+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:52:06.384+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:06.490+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.329 seconds
[2025-03-27T00:52:36.820+0000] {processor.py:157} INFO - Started process (PID=2297) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:36.823+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:52:36.832+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:52:36.832+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:36.857+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:52:36.866+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:52:36.860+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:52:36.869+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:52:36.922+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.120 seconds
[2025-03-27T00:53:07.760+0000] {processor.py:157} INFO - Started process (PID=2322) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:07.765+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:53:07.770+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:53:07.770+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:07.802+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:53:07.812+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:53:07.805+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:53:07.813+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:07.875+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.125 seconds
[2025-03-27T00:53:38.806+0000] {processor.py:157} INFO - Started process (PID=2347) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:38.811+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:53:38.816+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:53:38.816+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:38.835+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:53:38.841+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:53:38.838+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:53:38.841+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:53:38.881+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.082 seconds
[2025-03-27T00:54:09.822+0000] {processor.py:157} INFO - Started process (PID=2372) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:09.824+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:54:09.828+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:54:09.828+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:09.847+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:54:09.852+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:54:09.850+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:54:09.852+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:09.901+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.089 seconds
[2025-03-27T00:54:41.023+0000] {processor.py:157} INFO - Started process (PID=2397) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:41.025+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:54:41.031+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:54:41.031+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:41.060+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:54:41.069+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:54:41.066+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:54:41.069+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:54:41.113+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.106 seconds
[2025-03-27T00:55:11.317+0000] {processor.py:157} INFO - Started process (PID=2427) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:11.325+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:55:11.327+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:55:11.327+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:11.353+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:55:11.360+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:55:11.359+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:55:11.361+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:11.408+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.112 seconds
[2025-03-27T00:55:42.690+0000] {processor.py:157} INFO - Started process (PID=2472) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:42.697+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:55:42.718+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:55:42.716+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:43.282+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:55:43.298+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:55:43.291+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:55:43.299+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:55:43.397+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.726 seconds
[2025-03-27T00:56:14.089+0000] {processor.py:157} INFO - Started process (PID=2502) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:14.098+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:56:14.105+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:56:14.104+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:14.154+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:56:14.160+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:56:14.157+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:56:14.160+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:14.251+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.225 seconds
[2025-03-27T00:56:45.157+0000] {processor.py:157} INFO - Started process (PID=2532) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:45.160+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:56:45.170+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:56:45.170+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:45.192+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:56:45.197+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:56:45.195+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:56:45.198+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:56:45.239+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.091 seconds
[2025-03-27T00:57:15.963+0000] {processor.py:157} INFO - Started process (PID=2557) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:15.965+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:57:15.970+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:57:15.970+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:15.989+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:57:15.995+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:57:15.992+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:57:15.995+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:16.033+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.083 seconds
[2025-03-27T00:57:46.889+0000] {processor.py:157} INFO - Started process (PID=2587) to work on /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:46.891+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/customers_postgres_to_bq.py for tasks to queue
[2025-03-27T00:57:46.899+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:57:46.899+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:46.919+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py:429 AirflowProviderDeprecationWarning: `exists_ok` parameter is deprecated, please use `if_exists`
[2025-03-27T00:57:46.924+0000] {logging_mixin.py:149} INFO - [2025-03-27T00:57:46.922+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/customers_postgres_to_bq.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/customers_postgres_to_bq.py", line 76, in <module>
    sql=f"SELECT * FROM {SOURCE_SCHEMA}.{SOURCE_TABLE_NAME};",
NameError: name 'SOURCE_SCHEMA' is not defined
[2025-03-27T00:57:46.924+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/customers_postgres_to_bq.py
[2025-03-27T00:57:46.968+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/customers_postgres_to_bq.py took 0.092 seconds
